### ISSUE TITLE
Accessibility audit

### ISSUE THREAD

**Issue by mmontesanonyc** [nychealth/EH-dataportal#1153]
Let's audit portal content for:
- alt text on images (specifically data stories and data features, though we can also address each Key Topic page)
- accessible tables accompanying data vis (in data stories and data features)
- accessible links (in all content)

Order of operations:
- [ ] Create an audit spreadsheet with content as rows, and columns for issues to address
- [ ] Then, we can divide and conquer to implement the above

One of the big lifts here will be inserting accessible tables into each data story and data feature. For each visualization, we'll need to duplicate it, create an accessible table, and embed it in an accessible way; that will take a bit of additional code which we can collaborate on when we get to that step.

**Comment by mmontesanonyc** [comment]
@etorem, please remind me - are we waiting on my review of other data stories?

**Comment by etorem** [comment]
@mmontesanonyc yes, I updated the excel EHDP Accessibility Audit with which ones you've already looked at. If there are issues you are noticing in a handful at a time, feel free to pause, I will fix them throughout the remaining stories, and then resume.

**Comment by mmontesanonyc** [comment]
Reviewing data story accessibility improvements on #1234

**Comment by mmontesanonyc** [comment]
FYI @etorem I handled a few of the DF interactives recently, and just merged #1234, so I think we can close this! TY for all your work.

Worth keeping in mind that moving forward - for data stories and data features - we should make it a priority to implement these strategies. Additionally, for data features, reports, and things like that, I want to make sure that we have clear data downloads and 'about the data' type sections at the footer with consistent design.
